eppo_core/src/ufc/eval_details.rs

```diff
@@ -0,0 +1,102 @@
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+
+use crate::{AttributeValue, Attributes};
+
+use super::{AssignmentValue, Condition, FlagEvaluationError, Shard, Value};
+
+/// Details about feature flag evaluation.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct EvalFlagDetails {
+    pub flag_key: String,
+    pub subject_key: String,
+    pub subject_attributes: Attributes,
+    /// Timestamp when the flag was evaluated.
+    pub timestamp: DateTime<Utc>,
+    /// Details of configuration used for evaluation. None if configuration hasn't been fetched yet.
+    pub configuration_details: Option<ConfigurationDetails>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub result: Option<AssignmentValue>,
```

Comment:
Added this field in addition to `variation_value` because there's one final conversion from variation's `Value` to `AssignmentValue` which may fail.